pkg/controller/internalreleaseimage/internalreleaseimage_controller_test.go

```
@@ -217,6 +217,91 @@ func TestInternalReleaseImageStatusOnError(t *testing.T) {
+					"htpasswd should not change when already matching")
+			}
+		})
+	}
+}
+
+func mustGenerateHtpasswd(t *testing.T, password string) string {
+	t.Helper()
+	entry, err := generateHtpasswdEntry(ctrlcommon.IRIRegistryUsername, password)
+	assert.NoError(t, err)
+	return entry
+}
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)